packages/vite/src/node/constants.ts

```diff
@@ -11,8 +11,9 @@ export const DEFAULT_MAIN_FIELDS = [
   'jsnext'
 ]
 
-// Support browserslist
-// "defaults and supports es6-module and supports es6-module-dynamic-import",
+// Baseline support browserslist
+// "defaults and supports es6-module and supports es6-module-dynamic-import"
+// Higher browser versions may be needed for extra features.
```

Comment:
Browserlist can't query `import.meta` support so thought I'd add this here. Also because we don't follow this browserlist strictly, as we bump base on usage.